llvm/test/CodeGen/AArch64/fp-maximumnum-minimumnum.ll

```diff
@@ -0,0 +1,125 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+; RUN: llc --mtriple=aarch64 --mattr=+fullfp16 < %s | FileCheck %s --check-prefix=AARCH64
+
+define <2 x double> @max_v2f64(<2 x double> %a, <2 x double> %b) {
```

Comment:
All of these tests have nnan, but nnan is not mentioned in the name. need nnan and no-nnan variants of the tests